### PR TITLE
Update profile photo permission table

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

File: api-reference/beta/api/profilephoto_get.md [hunk 1]
```diff
@@ -4,28 +4,27 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (**profilePhoto** properties).
 
-A GET photo operation first looks for the specified photo in the user's mailbox on Exchange Online, and if 
-it's not available there, then looks in Azure Active Directory (AAD).
+A GET photo operation first attempt sto retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory (AAD).
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md]: typo.  Also I don't think we are allowed to abbreviate to AAD.  I believe Azure AD is OK.

File: api-reference/beta/api/profilephoto_get.md [hunk 2]
```diff
@@ -4,28 +4,27 @@
...
 
-The supported sizes of HD photos on Exchange Online are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
-'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in AAD.
+The supported sizes of HD photos on Office 365 are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
+'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in Azure Active Directory.
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md:10]: I think you should ask Kostas (Konstantin Seleskerov <koselesk@microsoft.com>) and Priya (Priya Venkatraman <prvenk@microsoft.com>) to review.  Kostas knows much better how this API works with regards to Office.  AAD stores just one photo, and doesn't do any rescaling/resizing when reading it back.  I don't know what the photos (different to the photo) API does if the photo is ONLY in AAD - Priya know this better.

File: api-reference/beta/api/profilephoto_get.md [hunk 3]
```diff
@@ -4,28 +4,27 @@
...
-however, supports only the user's work or school mailboxes and not personal mailboxes.
-
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
 
-*	Profile photo of any user in the tenant including the signed-in user - User.ReadBasic.All, User.Read.All, User.ReadWrite.All
-*	Profile photo of specifically the signed-in user - User.Read, User.ReadWrite, User.ReadBasic.All, User.Read.All, User.ReadWrite.All
-* Profile photo of a **group** - Group.Read.All, Group.ReadWrite.All
-* Photo of a **contact** - Contacts.Read, Contacts.ReadWrite
+> **Note:** The GET photo operation in beta supports a user's work, school, or personal mailboxes. The GET photo metadata operation, however, supports only the user's work or school mailboxes and not personal mailboxes.
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md]: We should say `accounts` not `mailboxes`.

File: api-reference/beta/api/profilephoto_get.md [hunk 4]
```diff
@@ -4,28 +4,27 @@
...
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | For **user** resource:<br/>User.ReadBasic.All, User.Read.All, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.Read.All, Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.Read, Contacts.ReadWrite |
+|Delegated (personal Microsoft account) <br /> **Note**: Metadata operation is not supported. | For **user** resource:<br/>User.ReadBasic.All, User.Read.All, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.Read.All, Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.Read, Contacts.ReadWrite |
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md]: The personal stuff is ALL wrong.
**If** it works (and I'm not sure it does currently), the only relevant permissions are User.Read, User.ReadWrite and Contacts.Read, Contacts.ReadWrite.  Groups are not supported in consumer scenarios.

File: api-reference/beta/api/profilephoto_get.md [hunk 5]
```diff
@@ -4,28 +4,27 @@
...
 
-*	Profile photo of any user in the tenant including the signed-in user - User.ReadBasic.All, User.Read.All, User.ReadWrite.All
-*	Profile photo of specifically the signed-in user - User.Read, User.ReadWrite, User.ReadBasic.All, User.Read.All, User.ReadWrite.All
-* Profile photo of a **group** - Group.Read.All, Group.ReadWrite.All
-* Photo of a **contact** - Contacts.Read, Contacts.ReadWrite
+> **Note:** The GET photo operation in beta supports a user's work, school, or personal mailboxes. The GET photo metadata operation, however, supports only the user's work or school mailboxes and not personal mailboxes.
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | For **user** resource:<br/>User.ReadBasic.All, User.Read.All, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.Read.All, Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.Read, Contacts.ReadWrite |
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md]: Missing User.Read and User.ReadWrite as least privileged.

File: api-reference/beta/api/profilephoto_get.md [hunk 6]
```diff
@@ -4,28 +4,27 @@
...
+|Delegated (personal Microsoft account) <br /> **Note**: Metadata operation is not supported. | For **user** resource:<br/>User.ReadBasic.All, User.Read.All, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.Read.All, Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.Read, Contacts.ReadWrite |
+|Application                        | For **user** resource:<br/>User.ReadBasic.All, User.Read.All, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.Read.All, Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.Read, Contacts.ReadWrite |
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md]: User.ReadBasic.All is a delegated permission only, I believe.  Please remove it.

File: api-reference/beta/api/profilephoto_update.md [hunk 7]
```diff
@@ -13,9 +13,11 @@ Use only PUT for this operation in the beta version.
 ## Permissions
 One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
 
-- Profile photo of the signed-in **user** - User.ReadWrite, User.ReadWrite.All
-- Profile photo of a **group** - Group.ReadWrite.All
-- Photo of a **contact** - Contacts.ReadWrite
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account)     | Profile photo of the signed-in **user**:<br/>User.ReadWrite, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.ReadWrite |
+|Delegated (personal Microsoft account) | Profile photo of the signed-in **user**:<br/>User.ReadWrite, User.ReadWrite.All<br /><br />For **group** resource:<br />Group.ReadWrite.All<br /><br />For **contact** resource:<br />Contacts.ReadWrite |
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_update.md]: Remove User.ReadWrite.All and Group.ReadWrite.All - this is not relevant in consumer.  However not sure this scenario even works.

File: api-reference/beta/api/profilephoto_get.md (hunk 2 above)
Comment (onemorekostas) [reply api-reference/beta/api/profilephoto_get.md:10]: The statement above regarding sizes of the photo is correct.

File: api-reference/beta/api/profilephoto_get.md [hunk 8]
```diff
@@ -4,28 +4,27 @@
...
 
 You can get the metadata of the largest available photo, or specify a size to get the metadata for that photo size.
 If the size you request is not available, you can still get a smaller size that the user has uploaded and made available.
 For example, if the user uploads a photo that is 504x504 pixels, then all but the 648x648 size of photo will be available for download.
-If the specified size is not available in the user's mailbox or in AAD, the size of '1x1' is returned with the rest of 
+If the specified size is not available in the user's mailbox or in Azure Active Directory, the size of '1x1' is returned with the rest of 
 metadata.
 
-> **Note** The GET photo operation in beta supports a user's work, school, or personal mailboxes. The GET photo metadata operation,
-however, supports only the user's work or school mailboxes and not personal mailboxes.
```
Comment (onemorekostas) [inline api-reference/beta/api/profilephoto_get.md]: @dkershaw10 as far as I remember photos for users with their personal mailboxes are not yet on v1.0 and are only on beta endpoint. Therefore, this note should not be deleted from this page for now.

**Review by onemorekostas — APPROVED** [review]

File: api-reference/beta/api/profilephoto_get.md [hunk 9]
```diff
@@ -4,28 +4,27 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (**profilePhoto** properties).
 
-A GET photo operation first looks for the specified photo in the user's mailbox on Exchange Online, and if 
-it's not available there, then looks in Azure Active Directory (AAD).
+A GET photo operation first attempt to retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory.
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md:7]: attempts
I also wonder whether you should go further and say - "If the photo is not available in Office 365, or if the user or tenant is not licensed for Office 365,...

File: api-reference/beta/api/profilephoto_get.md [hunk 10]
```diff
@@ -4,28 +4,27 @@
...
-The supported sizes of HD photos on Exchange Online are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
-'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in AAD.
+The supported sizes of HD photos on Office 365 are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
+'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in Azure Active Directory.
 
 You can get the metadata of the largest available photo, or specify a size to get the metadata for that photo size.
 If the size you request is not available, you can still get a smaller size that the user has uploaded and made available.
 For example, if the user uploads a photo that is 504x504 pixels, then all but the 648x648 size of photo will be available for download.
-If the specified size is not available in the user's mailbox or in AAD, the size of '1x1' is returned with the rest of 
+If the specified size is not available in the user's mailbox or in Azure Active Directory, the size of '1x1' is returned with the rest of
```
Comment (dkershaw10) [inline api-reference/beta/api/profilephoto_get.md:15]: Missing a "the" at the end of the sentence?

File: api-reference/v1.0/api/profilephoto_get.md [hunk 11]
```diff
@@ -2,17 +2,29 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (profilePhoto properties).
 
-A GET operation looks for the specified photo in the user's mailbox on Exchange Online.
-
 > **Note** This operation in version 1.0 supports only a user's work or school mailboxes and not personal mailboxes.
 
+A GET photo operation first attempt to retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory.
```
Comment (dkershaw10) [inline api-reference/v1.0/api/profilephoto_get.md]: the fallback mechanism is not deployed in v1 yet.

**Pull request closed without merging**